packages/vscode-extension/src/debugging/DebugSession.ts

```diff
@@ -230,6 +235,10 @@ export class DebugSession implements Disposable {
     return Promise.race([resultPromise, timeout]).catch((_e) => false);
   }
 
+  public postMessage(data: any) {
+    this.jsDebugSession?.customRequest("RNIDE_postMessage", data);
+  }
```

Comment:
I feel like this method's name is too generic. It's supposed to be used specifically for the inspector bridge connection, right? Maybe something more like
```suggestion
  public dispatchToRadonAgent(data: any) {
    this.jsDebugSession?.customRequest("RNIDE_dispatchToRadonAgent", data);
  }
```
since it calls `radon_dispatch` on the runtime? Or
```suggestion
  public postInspectorBridgeMessage(data: any) {
    this.jsDebugSession?.customRequest("RNIDE_postInspectorBridgeMessage", data);
  }
```
If we wanted to be less specific about how it's implemented, and more about the intent.
 I don't know really.